### ISSUE TITLE
Not working in chrome and firefox, please fix it, thanks.

### ISSUE THREAD

**Issue by Menorah** [wassname/keywordshitter2#1]
Not showing advanced options when i click in "advanced" button, the same with "about" button.

**Comment by wassname** [comment]
I'm in chrome and it works for me. Could you give me more info? OS, version, installed extensions etc.

 Could you go to the page http://wassname.github.io/keywordshitter2/ then open devtools (cntrl+shift+I), click console, then try clicking the advanced button again. Once you've done that, error messages may appear in the devtools. If so could you copy it here? Thanks

**Comment by Menorah** [comment]
Problem solved, had installed a copy on my server and not loading all java script files. Now works fine.

On the other hand, I can't make it work the "Copy Keywords" option from export button, only an **empty csv file** is downloaded.  

Would it be possible to export **only the keywords** in a archive.txt as keywordshitter.com makes it originally?

What about the possibility of adding negative and positive word filters and the option to filter the repeated results ?

Greetings and thanks for answering !

**Comment by wassname** [comment]
Oh nice to hear you solved it!

Yeah good point. Exporting as a simple list of keywords is something I want too. I'll put it as an bug for when I have time/motivation.

I think your the first person who has liked it enough to use it in anger. So do you have other ideas about what critical changes should be made? It would be good to hear your perspective. You know the feature/bug that make the difference between using it or not.

**Comment by Menorah** [comment]
Hello again, I have found a few small bugs:
1. bug:
   Sometimes keywordshitter2 freezes when making several requests and takes a bit to update the table of results.

2 bug
When I choose to copy the key words with the button export, copy all of the data (keywords+domain + id)

Example:
"30 hello world lyrics suggestqueries.google. com & ds = yt 5 115 android tutorial studio - 01 - hello world application..."

3 bug
"Datatables warning" pop-ups when charge the cache

See [here](http://imgur.com/a/3I4Jd).

Possible characteristics to be added:

Apply the service **google trends** (**trending graphic** in search of the main word, **regional interest** data, and **related searches** data)

Something like this?:

```
//Keywordtrending graphic.

GoogleTrendGraphkw: function (lang, kw){
  $('#google_trends').append('<iframe scrolling="no" style="border:none;" width="500" height="330" src="http://www.google.com/trends/fetchComponent?hl='+lang+'&q='+kw+'&cmpt=q&content=1&cid=TIMESERIES_GRAPH_0&export=5&w=500&h=330"></iframe>');
}

```

Ability to select more than one service at the same time (google, yandex, yahoo, etc) to perform the search.

Positive and negative words filters.

See data such as number of monthly searches, competition, difficulty, and cpc.

Regards!!

**Comment by wassname** [comment]
Thanks for the feedback.
1. Can't fix this sadly. It seems to be a limitation of JSONP which I have to use to by bypass no-cors policies (I did remove the db caching though which may have been contribution.)
2. Bug, but I've fixed that now.
3. Bug: Ah that's so annoying, It kept happening to me and I though I had squashed it. It happens when empty data is displayed.

Google trends. That's a good idea. And multiple services and filters would be nice. I don't have a use or motivation for these personally so I probably wont spend time implementing them. Any one else can feel free to put in the work though.
